utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/SchemaVersionOption.scala

```diff
@@ -21,3 +26,14 @@ object SchemaVersionOption {
 case class ExistingSchemaVersion(version: Int) extends SchemaVersionOption
 
 case object LatestSchemaVersion extends SchemaVersionOption
+
+case class DynamicSchemaVersion(typ: JsonTypes) extends SchemaVersionOption
+
+sealed abstract class JsonTypes(val value: Int) extends IntEnumEntry
+
+object JsonTypes extends IntEnum[JsonTypes] {
+  val values = findValues
+
+  case object Json  extends JsonTypes(1)
+  case object Plain extends JsonTypes(2)
```

Comment:
Maybe if we put `Plain` and  `Json` together in enum the enum should not be named `JsonTypes`? Maybe `ContentTypes`?